draft-ietf-scitt-architecture.md

```diff
@@ -290,26 +311,26 @@ SCITT provides a standardized way to:
 
 ### Promotion of a Software Component by Multiple Entities
 
-A software component (e.g., a library) released by a certain original producer is becoming popular.
-The released software component is accompanied by a statement of authenticity (e.g., a detached signature).
+A software component (e.g., a library or software product) released by a trusted producer is common practice for both open-source and commercial offerings.
```

Comment:
I had to read this about 5 times before I could convince myself it makes grammatical sense, but I'm OK with it now. Won't risk resetting all the other ticks to request a clarification.

Reply:
nod. If we can come up with a suggestion, we can merge in an editorial update.